content/api/cli.md

```diff
@@ -1,8 +1,264 @@
 ---
-title: Using the CLI
+title: CLI
 ---
-> Cli call
-> Cli options
-> --config
+## Overview
 
-> see also [[Using the configuration]]
+`webpack` provides a command line interface(cli) to configure and interact with your build.
+This is mostly useful in case of early prototyping, profiling, writing npm scripts or personal customization of the build.
+For proper usage and easy distribution of this configuration, webpack can be configured with `webpack.config.js`.
+Any parameters sent to the CLI will map to a corresponding parameter in the config file.
+
+## Installation
+
+``` sh
+$ npm install webpack -g
+```
+
+The `webpack` command is now available globally.
```

Comment:
We would have a section for installation. This info would be best there right? And I could link to it from here.